pkg/combustion/custom.go

```diff
@@ -41,30 +41,52 @@ func configureCustomFiles(ctx *image.Context) ([]string, error) {
 
 func handleCustomFiles(ctx *image.Context) error {
 	fullFilesDir := generateComponentPath(ctx, filepath.Join(customDir, customFilesDir))
-	_, err := copyCustomFiles(fullFilesDir, ctx.CombustionDir, nil)
+	err := copyCustomFiles(fullFilesDir, ctx.CombustionDir)
 	return err
 }
 
 func handleCustomScripts(ctx *image.Context) ([]string, error) {
 	fullScriptsDir := generateComponentPath(ctx, filepath.Join(customDir, customScriptsDir))
 	executablePerms := fileio.ExecutablePerms
-	scripts, err := copyCustomFiles(fullScriptsDir, ctx.CombustionDir, &executablePerms)
+	scripts, err := copyCustomScripts(fullScriptsDir, ctx.CombustionDir, &executablePerms)
 	return scripts, err
 }
 
-func copyCustomFiles(fromDir, toDir string, filePermissions *os.FileMode) ([]string, error) {
+func copyCustomFiles(fromDir, toDir string) error {
+	if _, err := os.Stat(fromDir); os.IsNotExist(err) {
```

Comment:
Is this just a safety check, or is there an expected situation where we'd run into this and want to fail gracefully?

Reply:
This is the clean way to way to just check if the directory exists and proceed if it does